subprojects/shipkit/src/main/groovy/org/shipkit/internal/gradle/git/OpenPullRequest.java

```diff
@@ -0,0 +1,31 @@
+package org.shipkit.internal.gradle.git;
+
+public class OpenPullRequest {
+    private String sha;
+    private String ref;
+    private String url;
+
+    public String getSha() {
```

Comment:
Can you add Javadoc on the class and all the public methods? (examples of values in the getters are super useful). Given my understanding of the code, this class will be public at some point when we promote pull request features from incubating state.

When you add Javadoc, this code will be even better :)